<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWindow</class>
 <widget class="QMainWindow" name="ManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>491</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>201</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="addpersonpushButton">
         <property name="text">
          <string>添加账号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="checkPersonButton">
         <property name="text">
          <string>查看账号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="checkComroomButton">
         <property name="text">
          <string>查看机房</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cleanOrderButton">
         <property name="text">
          <string>清空预约</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="text">
          <string>注销登录</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>321</width>
        <height>161</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QWidget" name="widget_6" native="true">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>223</width>
          <height>100</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>学  号：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="addstulineEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_8" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>密  码：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="addstupasswordlineEdit">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QWidget" name="widget_4" native="true">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>223</width>
          <height>100</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>职工号：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="addtealineEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>密  码：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="addteapasswordlineEdit">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>166</width>
        <height>34</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="addSturadioButton">
         <property name="text">
          <string>添加学生</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="addTearadioButton">
         <property name="text">
          <string>添加老师</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="confirmButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitpushButton_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QWidget" name="widget_9" native="true">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>211</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="checkallstupushButton">
         <property name="text">
          <string>查看所有学生</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="checkallteapushButton">
         <property name="text">
          <string>查看所有老师</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCommandLinkButton" name="exitcommandLinkButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>250</y>
        <width>172</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
